# Dutch translation of nssbackup.
# Copyright (C) 2006 Mark Van den Borre
# This file is distributed under the same license as the sbackup package.
# Mark Van den Borre <mvdborre@gmail.com>, 2006.
#
#
msgid ""
msgstr ""
"Project-Id-Version: nssbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-08-15 16:39+0200\n"
"PO-Revision-Date: 2010-05-30 16:16+0000\n"
"Last-Translator: Jean-Peer Lorenz <Unknown>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-05-31 03:44+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: src/sbackup/backupproc.py:168 src/sbackup/pkginfo.py:63
#, fuzzy, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr "NSsbackup Daemon <%(login)s@%(hostname)s>"

#: src/sbackup/backupproc.py:172
#, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr ""

#: src/sbackup/backupproc.py:177 src/sbackup/core/profile_handler.py:279
msgid "No log file specified."
msgstr ""

#: src/sbackup/backupproc.py:182
msgid "Unable to find log file."
msgstr ""

#: src/sbackup/backupproc.py:248
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""
"Kritieke fout: Configuratiebestand voor het default profiel is niet "
"gevonden!\n"
"\n"
"Verder gaan met de overige profielen."

#: src/sbackup/backupproc.py:297
msgid "Backup was canceled by user."
msgstr ""

#: src/sbackup/backupproc.py:302
msgid "An error occurred during the cancellation of backup:"
msgstr ""

#: src/sbackup/backupproc.py:308
msgid "An error occurred during the backup:"
msgstr ""

#: src/sbackup/backupproc.py:325
msgid "Error when sending email:"
msgstr ""

#: src/sbackup/backupproc.py:356
msgid "The following error(s) occurred before:"
msgstr "De volgende fout(en) kwamen eerder voor:"

#: src/sbackup/backupproc.py:535
#, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr ""

#: src/sbackup/pkginfo.py:43
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""

#: src/sbackup/pkginfo.py:55
msgid "translator-credits"
msgstr ""
"Vertaald door\n"
"\n"
"Launchpad Contributions:\n"
"  Arien Lam https://launchpad.net/~h-a-lam\n"
"  Jean-Peer Lorenz https://launchpad.net/~peer.loz\n"
"  Taygeto https://launchpad.net/~taygeto"

#: src/sbackup/ar_backend/tar.py:108 src/sbackup/ar_backend/tar.py:160
#: src/sbackup/ar_backend/tar.py:206
msgid "Invalid archive type."
msgstr ""

#: src/sbackup/ar_backend/tar.py:327
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""

#: src/sbackup/ar_backend/tar.py:336 src/sbackup/ar_backend/tar.py:395
#: src/sbackup/core/profile_handler.py:289
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr "Permissies veranderen voor bestand '%s' is mislukt."

#: src/sbackup/ar_backend/tar.py:345
#, fuzzy, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr "foutmelding ontvangen bij '%(file)s' : %(error)s"

#: src/sbackup/ar_backend/tar.py:362
msgid "Launching TAR to make incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:388
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:389
msgid "Falling back to full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:431
msgid "Launching TAR to make a full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:495
msgid "Leading '/' from member names were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:497
msgid "Leading '/' from hard link targets were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:506
#, python-format
msgid "TAR returned a message: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:508
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:511
msgid "TAR has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:515
#, python-format
msgid "TAR returned a warning: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:517
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:520
msgid "TAR returned warnings but has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:524
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr ""

#: src/sbackup/ar_backend/tar.py:705
#, python-format
msgid "Unable to terminate backend process: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:738
msgid "Included"
msgstr "Ingesloten"

#: src/sbackup/ar_backend/tar.py:738
msgid "Not changed"
msgstr "Ongewijzigd"

#: src/sbackup/ar_backend/tar.py:738
msgid "Directory"
msgstr "Map"

#: src/sbackup/ar_backend/tar.py:766
msgid "Line must be a string"
msgstr "Regel moet een string zijn"

#: src/sbackup/ar_backend/tar.py:782
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr "Dumpdir niet consistent : 'filename' is leeg"

#: src/sbackup/ar_backend/tar.py:795
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr "Dumpdir niet consistent : 'control' is leeg"

#: src/sbackup/ar_backend/tar.py:868
#, python-format
msgid "File '%s' does not exist."
msgstr ""

#: src/sbackup/ar_backend/tar.py:998 src/sbackup/ar_backend/tar.py:1032
#: src/sbackup/ar_backend/tar.py:1129
msgid "The snarfile header is incomplete."
msgstr ""

#: src/sbackup/ar_backend/tar.py:1298
msgid "A SnapshotFile is required"
msgstr "Een SnapshotFile is vereist"

#: src/sbackup/ar_backend/tar.py:1398
#, python-format
msgid "Directory does not exist: %s."
msgstr ""

#: src/sbackup/core/ConfigManager.py:273
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:276
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:422
msgid "Option 'target' is missing, aborting."
msgstr "Optie 'doel' mist, bezig met afbreken"

#: src/sbackup/core/ConfigManager.py:524 src/sbackup/core/ConfigManager.py:558
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""

#: src/sbackup/core/ConfigManager.py:550
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""
"Kon '%(parameter)s' niet als dictionary interpreteren (waarde = '%(value)r' )"

#: src/sbackup/core/ConfigManager.py:574
msgid "You must provide a dictionary."
msgstr ""

#: src/sbackup/core/ConfigManager.py:691
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:698
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr ""

#: src/sbackup/core/ConfigManager.py:717
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""

#: src/sbackup/core/ConfigManager.py:737
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr ""

#: src/sbackup/core/ConfigManager.py:749
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""
"sectie [%(section)s] in '%(configfile)s' hoort niet te bestaan, bezig met "
"afbreken"

#: src/sbackup/core/ConfigManager.py:815
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""

#: src/sbackup/core/ConfigManager.py:909 src/sbackup/core/ConfigManager.py:921
msgid "The config file is not set yet into this ConfigManager"
msgstr "Het configuratiebestand is nog niet ingesteld in deze ConfigManager"

#: src/sbackup/core/ConfigManager.py:1081
msgid "No receiver set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1083
msgid "No SMTP server set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1086
msgid "Username set but no password specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1089
msgid "Password set but no username specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1092
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1096
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1287
msgid "Default Profile"
msgstr "Standaardprofiel"

#: src/sbackup/core/ConfigManager.py:1288
msgid "Unknown Profile"
msgstr "Onbekend profiel"

#: src/sbackup/core/ConfigManager.py:1338
msgid "Unlimited"
msgstr "Onbeperkt"

#: src/sbackup/core/ConfigManager.py:1339
msgid "100 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1340
msgid "250 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1341
msgid "650 MiB"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1342
msgid "2 GiB (FAT16)"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1343
msgid "4 GiB (FAT32)"
msgstr ""

#: src/sbackup/core/ConfigManager.py:1344 src/sbackup/ui/configgui.py:275
#: src/sbackup/ui/configgui.py:1386
msgid "Custom"
msgstr "Aangepast"

#: src/sbackup/core/ConfigManager.py:1346
msgid "MP3 Music"
msgstr "MP3-muziek"

#: src/sbackup/core/ConfigManager.py:1347
msgid "AVI Video"
msgstr "AVI-video"

#: src/sbackup/core/ConfigManager.py:1348
#: src/sbackup/core/ConfigManager.py:1349
msgid "MPEG Video"
msgstr "MPEG-video"

#: src/sbackup/core/ConfigManager.py:1350
msgid "Matrjoshka Video"
msgstr "Matrjoshka-video"

#: src/sbackup/core/ConfigManager.py:1351
msgid "OGG Multimedia container"
msgstr "OGG-multimediacontainer"

#: src/sbackup/core/ConfigManager.py:1352
msgid "CD Images"
msgstr "CD-imagebestanden"

#: src/sbackup/core/RestoreManager.py:79
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr "Bestand '%s' niet gevonden in de backup bestandenlijst"

#: src/sbackup/core/RestoreManager.py:81
#, fuzzy, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""
"Bestand '%(filename)s' niet gevonden in de bestandenlijst van backup "
"snapshot [%(snapshotname)s], we gaan door!"

#: src/sbackup/core/SnapshotManager.py:125
#, python-format
msgid "Snapshot '%s' not found "
msgstr "Snapshot '%s' niet gevonden "

#: src/sbackup/core/SnapshotManager.py:216
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:219
#, fuzzy, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr "Ongeldig snapshot '%(name)s' , bezig met verwijderen: %(error_cause)s "

#: src/sbackup/core/SnapshotManager.py:309
#, python-format
msgid "No need to rebase a full snapshot '%s'"
msgstr "Een volledig snapshot '%s' hoeft niet op een ander geënt te worden"

#: src/sbackup/core/SnapshotManager.py:311
#, python-format
msgid ""
"'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
msgstr ""
"'%(snapshotToRebase)s'  heeft geen 'base' bestand , wellicht is het niet in "
"orde "

#: src/sbackup/core/SnapshotManager.py:316
#, python-format
msgid ""
"Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= '%"
"(NewBaseSnapshot)s' "
msgstr ""
"Kan een snapshot niet op een eerdere versie enten: '%(snapshotToRebase)s' <= "
"'%(NewBaseSnapshot)s' "

#: src/sbackup/core/SnapshotManager.py:323
#, python-format
msgid "Rebasing '%(current)s' on '%(base)s'"
msgstr "Snapshot '%(current)s' wordt op '%(base)s' geënt"

#: src/sbackup/core/SnapshotManager.py:560
#, python-format
msgid "An error occurred when pulling snapshot '%s'."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:599
msgid "The snarfile header is incomplete !"
msgstr "De header van het SNAR-bestand is incompleet"

#: src/sbackup/core/SnapshotManager.py:751
#, python-format
msgid "Base of snapshot '%s' is not set."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:788
#, python-format
msgid "Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:828
#, python-format
msgid "Cancelling pull of snapshot '%s'"
msgstr "Afzien van het ophalen van snapshot '%s'"

#: src/sbackup/core/SnapshotManager.py:998
msgid ""
"Logarithmic purge is currently disabled. Please specify a cutoff purge "
"interval in your configuration instead."
msgstr ""

#: src/sbackup/core/UpgradeManager.py:172
#, python-format
msgid "Upgrading from v1.0 to v1.2: %s"
msgstr "Bezig met upgraden van v1.0 naar v1.2: %s"

#: src/sbackup/core/UpgradeManager.py:189
#, python-format
msgid "Upgrading to v1.3: %s"
msgstr "Upgraden naar v1.3: %s"

#: src/sbackup/core/UpgradeManager.py:214
#, python-format
msgid "Upgrading to v1.4: %s"
msgstr "Upgraden naar v1.4: %s"

#: src/sbackup/core/UpgradeManager.py:228
#, python-format
msgid "Upgrading to v1.5: %s"
msgstr "Upgraden naar v1.5: %s"

#: src/sbackup/core/UpgradeManager.py:282
#, python-format
msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
msgstr ""

#: src/sbackup/core/profile_handler.py:90
#, fuzzy
msgid "Preparation of backup process"
msgstr "Backupsessie wordt gestart"

#: src/sbackup/core/profile_handler.py:134
msgid ""
"There are snapshots stored in outdated snapshot formats. Please upgrade them "
"using 'Simple Backup-Restoration' if you want to use them."
msgstr ""

#: src/sbackup/core/profile_handler.py:143
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr ""

#: src/sbackup/core/profile_handler.py:151
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:160
msgid "Setting packages File."
msgstr ""

#: src/sbackup/core/profile_handler.py:166
msgid "Problem when setting the packages list: "
msgstr ""

#: src/sbackup/core/profile_handler.py:170
msgid "Setting Excludes File."
msgstr ""

#: src/sbackup/core/profile_handler.py:179
#, python-format
msgid "Setting compression format to `%s`"
msgstr ""

#: src/sbackup/core/profile_handler.py:186
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:193
msgid "Option 'Follow symbolic links' is enabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:195
msgid "Option 'Follow symbolic links' is disabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:204
#, fuzzy
msgid "Snapshot is being committed"
msgstr "Een SnapshotFile is vereist"

#: src/sbackup/core/profile_handler.py:218
#, python-format
msgid "Error while purging old snapshots: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:220
msgid "Backup process finished."
msgstr ""

#: src/sbackup/core/profile_handler.py:227
msgid "Inspect file system and collect file infos"
msgstr ""

#: src/sbackup/core/profile_handler.py:238
msgid "Summary of backup"
msgstr ""

#: src/sbackup/core/profile_handler.py:239
#, python-format
msgid "Number of directories: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:240
#, python-format
msgid "Total number of files: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of symlinks: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of new files (also included): %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:246
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:247
#, python-format
msgid "Maximum free size required is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:253
#, python-format
msgid "Available disk size is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:255
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (%"
"(freespace)s <= %(neededspace)s)."
msgstr ""

#: src/sbackup/core/profile_handler.py:274
msgid "Unable to copy log. File access is not initialized."
msgstr ""

#: src/sbackup/core/profile_handler.py:292
#, fuzzy, python-format
msgid "Unable to copy log file: %s"
msgstr "Geen snapshot om logbestand naar te kopieren."

#: src/sbackup/core/profile_handler.py:294
msgid "Unable to find logfile to copy into snapshot."
msgstr ""

#: src/sbackup/core/profile_handler.py:296
msgid "No snapshot to copy logfile."
msgstr "Geen snapshot om logbestand naar te kopieren."

#: src/sbackup/core/profile_handler.py:303
msgid "Processing of profile was canceled on user request\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:320
msgid "Processing of profile successfully finished (no errors)\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:326
#, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:337
#, fuzzy, python-format
msgid "Backup destination: %s"
msgstr "NSsbackup herstel"

#: src/sbackup/core/profile_handler.py:341
#, fuzzy
msgid "Unable to find destination directory."
msgstr "Doelmap kiezen"

#: src/sbackup/core/profile_handler.py:360
msgid "Unable to find destination directory even after retry."
msgstr ""

#: src/sbackup/core/profile_handler.py:361
#: src/sbackup/core/profile_handler.py:368
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr ""

#: src/sbackup/core/profile_handler.py:374
#, python-format
msgid "Unable to access destination: %s"
msgstr ""

#: src/sbackup/core/snapshot.py:129
msgid "Snapshot is inconsistent: __name is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:140
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr "Naam van snapshot '%s' voldoet niet aan vereiste"

#: src/sbackup/core/snapshot.py:234
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:321
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""

#: src/sbackup/core/snapshot.py:345
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""

#: src/sbackup/core/snapshot.py:482
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr "Ongeldige naam van snapshot : %s"

#: src/sbackup/core/snapshot.py:500
#, python-format
msgid "Name of base not valid : %s"
msgstr "Ongeldige naam voor base : %s"

#: src/sbackup/core/snapshot.py:546
#, fuzzy, python-format
msgid "Invalid name of snapshot: %s"
msgstr "Afzien van het ophalen van snapshot '%s'"

#: src/sbackup/core/snapshot.py:550
msgid "Unable to read mandatory file `ver`"
msgstr ""

#: src/sbackup/core/snapshot.py:595
msgid "Base name must be set for incremental backup."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:44
#: src/sbackup/fs_backend/_gio_utils.py:106
msgid "SFTP"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:107
msgid "SSH"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:108
msgid "FTP"
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:96
msgid "Initializing GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:110
msgid "Terminating GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "NFS"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:187
msgid "Unable to get mount: path is not mounted"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:189
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:191
#: src/sbackup/fs_backend/_gio_utils.py:194
#, fuzzy, python-format
msgid "Unable to get mount: %s"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/fs_backend/_gio_utils.py:241
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:355
msgid "Path is already mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:378
msgid "Path is not mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:809
msgid "test specified path for existence using GIO"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:822
#, fuzzy
msgid "Create testdir"
msgstr "Bezig met aanmaken doelmap '%s'"

#: src/sbackup/fs_backend/_gio_utils.py:827
msgid "Test testfile for existence"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:834
#, fuzzy
msgid "Create file"
msgstr "Standaardprofiel"

#: src/sbackup/fs_backend/_gio_utils.py:841
msgid "Re-read test file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:853
msgid "Remove file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:855
#, fuzzy
msgid "Remove dir"
msgstr "Terugdraaien"

#: src/sbackup/fuse_plugins/__init__.py:160
#, fuzzy, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""
"Importeren van plugin %(plugin_name)s is mislukt! Oorzaak: %(error_cause)s "

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""
"Externe applicatie 'curlftpfs', vereist voor ftp sites, niet gevonden: %s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr "Mounten van '%(server)s' in '%(mountpoint)s'  mislukt: %(error)s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Fout bij het matchen van '%s' met het schema 'ftp://user:pass@server/"
"anything'  (de '/' na server is verplicht)"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr "SFTP schema is als: sftp://user:pass@server/anything"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Fout bij het matchen van '%s' met het schema 'sftp://user:pass@server/"
"anything'  (de '/' na server is verplicht)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Fout bij het matchen van '%s' met het schema 'ssh://user:pass@example.com/"
"home/'  (de '/' na server is verplicht)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => %"
"(erroroutput)s "
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%"
"s' (The '/' after server is mandatory)"
msgstr ""

#: src/sbackup/ui/GladeWindow.py:201
#, fuzzy
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""
"Zorg ervoor dat de missende map bestaat (door bijvoorbeeld een externe "
"schijf te mounten) of verander het doel naar een bestaande map."

#: src/sbackup/ui/GladeWindow.py:202
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:203 src/sbackup/ui/restoregui.py:783
#: src/sbackup/ui/restoregui.py:837
#, fuzzy
msgid "Simple Backup error"
msgstr "NSsbackup foutmelding"

#: src/sbackup/ui/GladeWindow.py:204
msgid "Unable to access backup destination"
msgstr ""

#: src/sbackup/ui/configgui.py:121
msgid "Scheduled backups are available for Administrator users only."
msgstr ""

#: src/sbackup/ui/configgui.py:131 src/sbackup/ui/configgui.py:140
msgid "Name"
msgstr "Naam"

#: src/sbackup/ui/configgui.py:148
msgid "File Type"
msgstr "Bestandstype"

#: src/sbackup/ui/configgui.py:183
msgid "Enable"
msgstr "Inschakelen"

#: src/sbackup/ui/configgui.py:184
msgid "Profile Name"
msgstr "Profielnaam"

#: src/sbackup/ui/configgui.py:281
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""

#: src/sbackup/ui/configgui.py:288
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:535
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""
"Zorg ervoor dat de missende map bestaat (door bijvoorbeeld een externe "
"schijf te mounten) of verander het doel naar een bestaande map."

#: src/sbackup/ui/configgui.py:536
msgid ""
"Backup target does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:538
msgid "Unable to open backup target"
msgstr ""

#: src/sbackup/ui/configgui.py:609
#, python-format
msgid "Current profile: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:700
msgid "Already included item !"
msgstr "Onderdeel al opgenomen!"

#: src/sbackup/ui/configgui.py:715
msgid "Already excluded item !"
msgstr "Onderdeel al uitgesloten!"

#: src/sbackup/ui/configgui.py:727 src/sbackup/ui/configgui.py:1426
msgid "Empty expression. Please enter a valid regular expression."
msgstr "Lege expressie. Voer alstublieft een geldige regular expression in."

#: src/sbackup/ui/configgui.py:742 src/sbackup/ui/configgui.py:1440
msgid "Provided regular expression is not valid."
msgstr "Ongeldige regular expression ingevoerd."

#: src/sbackup/ui/configgui.py:750
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Lege bestandsnaam of pad. Voer alstublieft een geldige bestandsnaam of pad "
"in."

#: src/sbackup/ui/configgui.py:759
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""

#: src/sbackup/ui/configgui.py:788
msgid "Save configuration as..."
msgstr ""

#: src/sbackup/ui/configgui.py:855
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""

#: src/sbackup/ui/configgui.py:856
msgid "Backup process started"
msgstr ""

#: src/sbackup/ui/configgui.py:861 src/sbackup/ui/configgui.py:1629
#: src/sbackup/ui/configgui.py:1999
#, python-format
msgid "Unexpected dialog response: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:935
msgid "Include file..."
msgstr ""

#: src/sbackup/ui/configgui.py:942 src/sbackup/ui/configgui.py:997
msgid "All files"
msgstr "Alle bestanden"

#: src/sbackup/ui/configgui.py:962
msgid "Include folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:990
msgid "Exclude file..."
msgstr ""

#: src/sbackup/ui/configgui.py:1013
msgid "Exclude folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:1278
msgid "Configuration has changed"
msgstr ""

#: src/sbackup/ui/configgui.py:1279
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""

#: src/sbackup/ui/configgui.py:1285 src/sbackup/ui/configgui.py:1291
msgid "Test mail settings"
msgstr "Test mail instellingen"

#: src/sbackup/ui/configgui.py:1286
msgid "The test failed with following output:"
msgstr ""

#: src/sbackup/ui/configgui.py:1292
msgid "The test was successful."
msgstr ""

#: src/sbackup/ui/configgui.py:1375 src/sbackup/ui/configgui.py:1418
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""

#: src/sbackup/ui/configgui.py:1378 src/sbackup/ui/configgui.py:1421
msgid "Unsupported character"
msgstr ""

#: src/sbackup/ui/configgui.py:1589
#, fuzzy
msgid "Connect to remote destination"
msgstr "Doelmap kiezen"

#: src/sbackup/ui/configgui.py:1637
msgid "Mount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1648
#, fuzzy
msgid "Unable to mount host"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/configgui.py:1657
msgid "Umount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1667
#, fuzzy
msgid "Unable to unmount host"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/configgui.py:1697
msgid "Perfom tests on remote host"
msgstr ""

#: src/sbackup/ui/configgui.py:1710
msgid "Unable to access remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1714
msgid "All tests passed"
msgstr ""

#: src/sbackup/ui/configgui.py:1871
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""

#: src/sbackup/ui/configgui.py:1872 src/sbackup/ui/configgui.py:1879
msgid "Profile name not valid"
msgstr ""

#: src/sbackup/ui/configgui.py:1878
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""

#: src/sbackup/ui/configgui.py:1880
msgid "Renaming of profiles is not supported."
msgstr ""

#: src/sbackup/ui/configgui.py:1910
#, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""

#: src/sbackup/ui/configgui.py:1983
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: '%"
"s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:1993
msgid "Default settings are being applied."
msgstr ""

#: src/sbackup/ui/configgui.py:2018
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2032
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2045
#, fuzzy
msgid "Please select a profile."
msgstr "Selecteer een profiel!"

#: src/sbackup/ui/indicator_core.py:116
#, fuzzy
msgid "there was a problem initializing the pynotify module"
msgstr "er deed zich een probleem voor bij het initialieren van de "

#: src/sbackup/ui/indicator_core.py:145 src/sbackup/ui/indicator_core.py:192
#: src/sbackup/ui/indicator_core.py:220 src/sbackup/ui/indicator_core.py:242
#, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:305
msgid "Simple Backup"
msgstr ""

#: src/sbackup/ui/indicator_core.py:311
msgid "Show message windows"
msgstr ""

#: src/sbackup/ui/indicator_core.py:353 src/sbackup/ui/indicator_core.py:807
#, fuzzy
msgid "Cancel Backup"
msgstr "Oude backups opruimen"

#: src/sbackup/ui/indicator_core.py:415
msgid "Connection to D-Bus service lost."
msgstr ""

#: src/sbackup/ui/indicator_core.py:440
msgid "re-connection was successful"
msgstr ""

#: src/sbackup/ui/indicator_core.py:479
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""

#: src/sbackup/ui/indicator_core.py:508
#, fuzzy
msgid "Starting backup session"
msgstr "Backupsessie wordt gestart"

#: src/sbackup/ui/indicator_core.py:522
#, fuzzy
msgid "Ending Backup Session"
msgstr "Backupsessie wordt gestart"

#: src/sbackup/ui/indicator_core.py:535
msgid ""
"There are snapshots with old snapshot format. Please upgrade these using the "
"Restoration tool if you want to use them."
msgstr ""

#: src/sbackup/ui/indicator_core.py:537
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr ""

#: src/sbackup/ui/indicator_core.py:578
msgid "Try again"
msgstr ""

#: src/sbackup/ui/indicator_core.py:642
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:652
#, python-format
msgid "Unable to present window: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:668
#, fuzzy, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr "Wert je zeker dat je '%(current)s' op '%(base)s' wil enten?"

#: src/sbackup/ui/indicator_core.py:672
#, fuzzy
msgid "Cancel running backup process?"
msgstr "Backupsessie wordt gestart"

#: src/sbackup/ui/indicator_core.py:689
msgid "Unable to cancel backup process."
msgstr ""

#: src/sbackup/ui/indicator_core.py:751
#, fuzzy, python-format
msgid "Profile: %s"
msgstr "Profielen"

#: src/sbackup/ui/indicator_core.py:752
#, python-format
msgid "Size of backup: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:753
#, python-format
msgid "%.1f%% processed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:754
#, python-format
msgid "Remaining time: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:800
#, python-format
msgid "PID of backup process being canceled: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:805
#, python-format
msgid "Unable to send signal to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:806
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:810
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:845
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""

#: src/sbackup/ui/indicator_core.py:846
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The profile execution is canceled automatically in %s seconds."
msgstr ""

#: src/sbackup/ui/indicator_core.py:872 src/sbackup/ui/indicator_core.py:914
#: src/sbackup/ui/indicator_core.py:932 src/sbackup/ui/indicator_core.py:940
#: src/sbackup/ui/indicator_core.py:957
msgid "unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:893
msgid "less than 1 minute"
msgstr ""

#: src/sbackup/ui/indicator_core.py:895
msgid "about 2 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:897
msgid "about 3 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:899
msgid "about 4 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:901
msgid "about 5 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:903
#, python-format
msgid "about %.0f minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:913
msgid "In progress"
msgstr ""

#: src/sbackup/ui/indicator_core.py:923 src/sbackup/ui/indicator_core.py:949
msgid "finished"
msgstr ""

#: src/sbackup/ui/indicator_core.py:931
msgid "Progress: unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:938
#, fuzzy
msgid "Backup is being prepared"
msgstr "BackupManager gecreëerd "

#: src/sbackup/ui/indicator_core.py:939
msgid "Collecting file info"
msgstr ""

#: src/sbackup/ui/indicator_core.py:948
msgid "Progress: canceled"
msgstr ""

#: src/sbackup/ui/indicator_core.py:968
msgid "An error occurred"
msgstr ""

#: src/sbackup/ui/indicator_core.py:1001
#, fuzzy
msgid "Another `Simple Backup Indicator` is already running."
msgstr "Een andere NSsbackup daemon is al bezig (pid = %s )!"

#: src/sbackup/ui/indicator_core.py:1025
#, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr ""

#: src/sbackup/ui/misc.py:293
#, python-format
msgid "%s   (Administrator mode)"
msgstr ""

#: src/sbackup/ui/misc.py:323
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""

#: src/sbackup/ui/misc.py:324
msgid "Sorry, this should not have happened"
msgstr ""

#: src/sbackup/ui/restoregui.py:77
msgid "Restore..."
msgstr "Herstellen..."

#: src/sbackup/ui/restoregui.py:78 data/ui/sbackup-restore.glade:845
msgid "Restore as..."
msgstr "herstellen als ..."

#: src/sbackup/ui/restoregui.py:79
msgid "Revert..."
msgstr "Terugzetten ..."

#: src/sbackup/ui/restoregui.py:80 data/ui/sbackup-restore.glade:877
msgid "Revert as..."
msgstr "Terugzetten als ..."

#: src/sbackup/ui/restoregui.py:127 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:591
msgid "n.a."
msgstr ""

#: src/sbackup/ui/restoregui.py:145 src/sbackup/ui/restoregui.py:162
msgid "Snapshots"
msgstr "Snapshots"

#: src/sbackup/ui/restoregui.py:146
msgid "Version"
msgstr "Versie"

#: src/sbackup/ui/restoregui.py:153 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Pad"

#: src/sbackup/ui/restoregui.py:158
msgid "State"
msgstr "Toestand"

#: src/sbackup/ui/restoregui.py:177
#, fuzzy
msgid "Connect to destination..."
msgstr "Doelmap kiezen"

#: src/sbackup/ui/restoregui.py:201
msgid "Specified path does not exist"
msgstr ""

#: src/sbackup/ui/restoregui.py:246
msgid "Ready"
msgstr "Gereed"

#: src/sbackup/ui/restoregui.py:360
msgid "No destination specified"
msgstr ""

#: src/sbackup/ui/restoregui.py:377
msgid "Choose a source folder"
msgstr "Kies een bronmap"

#: src/sbackup/ui/restoregui.py:394
#, fuzzy
msgid "Reading snapshots..."
msgstr "Bezig backup snapshot te lezen ..."

#: src/sbackup/ui/restoregui.py:445 src/sbackup/ui/restoregui.py:483
msgid "Loading ..."
msgstr "Bezig met laden ..."

#: src/sbackup/ui/restoregui.py:532
#, fuzzy
msgid "Simple Backup restore error"
msgstr "NSsbackup herstel-fout"

#: src/sbackup/ui/restoregui.py:533
msgid "An error occurred while reading snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:538
msgid "This snapshot seems empty."
msgstr "Dit snapshot schijnt leeg te zijn."

#: src/sbackup/ui/restoregui.py:555
msgid "Reading backup snapshot..."
msgstr "Bezig backup snapshot te lezen ..."

#: src/sbackup/ui/restoregui.py:578
msgid "No backups found for this day."
msgstr ""

#: src/sbackup/ui/restoregui.py:593
#, fuzzy, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found. You should upgrade it. "
msgstr ""
"De snapshot-versie wordt niet ondersteund ( Versie %(supportedversion)s "
"wordt ondersteund). Versie '%(currentversion)s' gevonden. Het is aan te "
"raden om een upgrade te doen. "

#: src/sbackup/ui/restoregui.py:620
msgid "Select restore location"
msgstr "Herstel-locatie selecteren"

#: src/sbackup/ui/restoregui.py:638
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:651
msgid "Select revert location"
msgstr "Terugzet-locatie selecteren"

#: src/sbackup/ui/restoregui.py:660
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:738
#, fuzzy
msgid "Reading snapshot..."
msgstr "Bezig backup snapshot te lezen ..."

#: src/sbackup/ui/restoregui.py:757
msgid "The version of the snapshot is greater than the supported one!"
msgstr "De versie van het snapshot is hoger dan wordt ondersteund!"

#: src/sbackup/ui/restoregui.py:767
#, python-format
msgid ""
"Upgrading the selected snapshot `%s` might corrupt your backup for this "
"particular point in time. You cannot undo this operation. It is recommended "
"to create a copy of the snapshot directory before upgrading it. The upgrade "
"process takes a while - be patient.\n"
"\n"
"Are you sure that you want to upgrade the snapshot now?"
msgstr ""

#: src/sbackup/ui/restoregui.py:771
#, fuzzy
msgid "Do you really want to upgrade snapshot?"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/restoregui.py:781
#, python-format
msgid ""
"While attempting to upgrade snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:784
msgid "Unable to upgrade snapshot"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/restoregui.py:804
#, python-format
msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
msgstr "Wert je zeker dat je '%(current)s' op '%(base)s' wil enten?"

#: src/sbackup/ui/restoregui.py:822
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""

#: src/sbackup/ui/restoregui.py:826
#, fuzzy
msgid "Do you really want to remove snapshot?"
msgstr "Wert je zeker dat je '%(current)s' op '%(base)s' wil enten?"

#: src/sbackup/ui/restoregui.py:835
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:838
#, fuzzy
msgid "Unable to delete snapshot"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/restoregui.py:844
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""

#: src/sbackup/ui/restoregui.py:846
#, fuzzy
msgid "Unable to remove snapshot"
msgstr "Upgrade van snapshot is mislukt"

#: src/sbackup/ui/restoregui.py:854
msgid "Termination in progress"
msgstr ""

#: src/sbackup/ui/restoregui.py:870 src/sbackup/ui/restoregui.py:876
#: src/sbackup/ui/restoregui.py:882 src/sbackup/ui/restoregui.py:888
#: data/ui/sbackup-restore.glade:1039
#, fuzzy
msgid "Simple Backup restoration"
msgstr "NSsbackup herstel"

#: src/sbackup/ui/restoregui.py:871 src/sbackup/ui/restoregui.py:877
msgid "<b>Restoring of selected files</b>"
msgstr "<b>Herstel van geselecteerde bestanden</b>"

#: src/sbackup/ui/restoregui.py:872
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr "Herstel van <tt>'%s'</tt> is nu bezig."

#: src/sbackup/ui/restoregui.py:873
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:874
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Herstel van <tt>'%s'</tt> is nu bezig."

#: src/sbackup/ui/restoregui.py:878
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Herstel van <tt>'%(source)s'</tt>\n"
"naar <tt>'%(dirname)s'</tt> is nu bezig."

#: src/sbackup/ui/restoregui.py:879
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:880
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Herstel van <tt>'%(source)s'</tt>\n"
"naar <tt>'%(dirname)s'</tt> is nu bezig."

#: src/sbackup/ui/restoregui.py:883 src/sbackup/ui/restoregui.py:889
msgid "<b>Reverting selected files</b>"
msgstr "<b>Bezig met terugzetten van geselecteerde bestanden</b>"

#: src/sbackup/ui/restoregui.py:884
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr "Terugzetten van <tt>'%s'</tt> is nu bezig\n"

#: src/sbackup/ui/restoregui.py:885
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:886
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Terugzetten van <tt>'%s'</tt> is nu bezig\n"

#: src/sbackup/ui/restoregui.py:890
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""

#: src/sbackup/ui/restoregui.py:891
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:892
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Herstel van <tt>'%(source)s'</tt>\n"
"naar <tt>'%(dirname)s'</tt> is nu bezig."

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr ""

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr ""

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr ""

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr ""

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' does not exist."
msgstr ""

#: src/sbackup/util/filecollect.py:348
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:364
#, python-format
msgid "File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""

#: src/sbackup/util/filecollect.py:368
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:385
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr ""

#: src/sbackup/util/filecollect.py:394
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""

#: src/sbackup/util/filecollect.py:414
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""

#: src/sbackup/util/filecollect.py:424
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:430
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr ""

#: src/sbackup/util/filecollect.py:501
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr ""

#: src/sbackup/util/filecollect.py:552
msgid "Empty regular expression found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:558
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:577
msgid "No directories to backup defined."
msgstr ""

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr ""

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr ""

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr ""

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
msgid "Unable to remove lock file: File not found."
msgstr ""

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Bestandstype uitsluiten"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Standaard bestandstypen:"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Aangepast bestandstype met "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "bestandsextensie"

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Uitsluiten op basis van reguliere expressie"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Reguliere expressie invoeren.\n"
"Bestanden en mappen die hieraan voldoen zullen van backup worden uitgesloten."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr "Voer het doel om in te sluiten"

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr "Kies een plugin voor het format"

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr "Kies een plugin :"

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr "Controleer"

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr ""

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr "<b>Configuratie profielen</b>"

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr ""

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr ""

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
msgid "Simple Backup Suite"
msgstr ""

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr ""

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr ""

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr ""

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr ""

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr ""

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr ""

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr ""

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr ""

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr ""

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr "Profielen"

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Volledige backup maken minstens om de"

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "dagen"

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr " "

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr ""

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr ""

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr ""

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr ""

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Algemeen"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr ""

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Bestand Toevoegen"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Map Toevoegen"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Insluiten"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Paden"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr ""

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Bestandstypen"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr ""

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr ""

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr "Andere"

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Uitsluiten"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr ""

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr ""

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Gebruik zelfgekozen lokale backup-map"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Doelmap kiezen"

#: data/ui/sbackup-config.glade:2526
#, fuzzy
msgid "Use a remote site"
msgstr "Bron op afstand toevoegen"

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://gebruikersnaam:wachtwoord@voorbeeld.com/map/op/afstand/"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr ""

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Bestemming"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr ""

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr ""

#: data/ui/sbackup-config.glade:2747
#, fuzzy
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""
"Bij deze optie maakt NSsbackup de backup niet op een exact tijdstip maar "
"wanneer de computer aanstaat. "

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr ""

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr ""

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr ""

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr ""

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr ""

#: data/ui/sbackup-config.glade:2955
#, fuzzy
msgid "Enable purging of old backups"
msgstr "Verwijderen van oude en onvolledige backups inschakelen"

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr ""

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr "Logaritmisch"

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday.\n"
"Keep one backup per day from last week.\n"
"Keep one backup per week from last month.\n"
"Keep one backup per month from last year.\n"
"Keep one backup per year further into past.\n"
"Erase all other backups.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr ""

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"<small>Noot: Daarenboven zullen alle onvolledige en onherstelbare backups  "
"verwijderd worden zolang er een recentere volledige backup is.</small>"

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Opruimen"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Oude backups opruimen"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr "Debug is alleen voor het debuggen. Info is de aanbevolen instelling"

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr ""

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr "Map voor logbestand"

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr ""

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr ""

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr ""

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr ""

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr "Gebruikersnaam en wachtwoord"

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr "Gebruikersnaam"

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr "Wachtwoord"

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr ""

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr ""

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr ""

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr "Certificaat"

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr ""

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr ""

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr ""

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr ""

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr ""

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr ""

#: data/ui/sbackup-config.glade:3935
#, fuzzy
msgid "Set remote target"
msgstr "Voer het doel om in te sluiten"

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr ""

#: data/ui/sbackup-config.glade:3997
#, fuzzy
msgid "Hostname"
msgstr "Gebruikersnaam"

#: data/ui/sbackup-config.glade:4027
#, fuzzy
msgid "<b>Details</b>"
msgstr "<b>Snapshot details</b>"

#: data/ui/sbackup-config.glade:4157
#, fuzzy
msgid "Show password"
msgstr "Wachtwoord"

#: data/ui/sbackup-config.glade:4217
#, fuzzy
msgid "_Connect"
msgstr "Inhoud"

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr ""

#: data/ui/sbackup-restore.glade:38
#, fuzzy
msgid "<b>Backup destination</b>"
msgstr "NSsbackup herstel"

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr ""

#: data/ui/sbackup-restore.glade:85
#, fuzzy
msgid "Default profile's destination"
msgstr "Standaardprofiel"

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""

#: data/ui/sbackup-restore.glade:137
#, fuzzy
msgid "Custom destination"
msgstr "Bestemming"

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""

#: data/ui/sbackup-restore.glade:310
#, fuzzy
msgid "Available backups"
msgstr "<b>Beschikbare backups</b>"

#: data/ui/sbackup-restore.glade:398
#, fuzzy
msgid "Delete snapshot"
msgstr "Huidige snapshot verwijderen?"

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr "Verwijder"

#: data/ui/sbackup-restore.glade:475
#, fuzzy
msgid "Upgrade snapshot"
msgstr "Upgrade van snapshot is mislukt"

#: data/ui/sbackup-restore.glade:515
msgid "Upgrade"
msgstr "Upgraden"

#: data/ui/sbackup-restore.glade:553
msgid "<b>Snapshot History</b>"
msgstr "<b>Snapshot Geschiedenis</b>"

#: data/ui/sbackup-restore.glade:579
msgid "Current base:"
msgstr ""

#: data/ui/sbackup-restore.glade:620
msgid "Select a snapshot on this list to rebase on it."
msgstr "Selecteer een snapshot in de lijst om op te enten."

#: data/ui/sbackup-restore.glade:695
msgid "Rebase"
msgstr ""

#: data/ui/sbackup-restore.glade:746
msgid "<b>Snapshot Management</b>"
msgstr "<b>Snapshot Beheer</b>"

#: data/ui/sbackup-restore.glade:785
#, fuzzy
msgid "Snapshot details"
msgstr "<b>Snapshot details</b>"

#: data/ui/sbackup-restore.glade:850 data/ui/sbackup-restore.glade:867
msgid "Restores files/directories from the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:862
msgid "Restore"
msgstr "Herstellen"

#: data/ui/sbackup-restore.glade:882 data/ui/sbackup-restore.glade:901
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:896
msgid "Revert"
msgstr "Terugdraaien"

#: data/ui/sbackup-restore.glade:939
msgid "<b>Restoration Management</b>"
msgstr ""

#: data/ui/sbackup-restore.glade:964
msgid "<b>Export Management</b>"
msgstr "<b>Export Beheer</b>"

#: data/ui/sbackup-restore.glade:1056
msgid "Headline"
msgstr "Koptekst"

#: data/ui/sbackup-restore.glade:1069
msgid "Content"
msgstr "Inhoud"

#: scripts/sbackup-upgrade-backups.py:45
msgid ""
"\n"
"Simple Backup suit command line backup format upgrade\n"
"Usage: upgrade-backup backup-target-url\n"
"Note: backup-target-url must not include the snapshot subdirectory name, for "
"example:\n"
"\n"
"   /var/backup/\n"
"\n"
"Use simple-restore-gnome for more ease of use.\n"
msgstr ""
"\n"
"Simple Backup suit command line backup format upgrade\n"
"Gebruik: upgrade-backup backup-doel-url\n"
"Noot: backup-doel-url moet niet de snapshot subdirectorynaam bevatten, "
"bijv.:\n"
"\n"
"   /var/backup/\n"
"\n"
"Gebruik simple-restore-gnome voor meer gebruiksgemak.\n"

#~ msgid "Invalid value of the target directory : "
#~ msgstr "Ongeldige aanduiding van de doelmap: "

#~ msgid ""
#~ "Got a non valid snapshot '%(name)s' due to name convention : %"
#~ "(error_cause)s "
#~ msgstr ""
#~ "Snapshot '%(name)s' is ongeldig vanwege naamgevingsregels: %(error_cause)"
#~ "s "

#~ msgid "'%d' test(s) didn't succeed !"
#~ msgstr "'%d' test(s) mislukt !"

#~ msgid ""
#~ "Please test writability of the target directory by pressing \"Test\" "
#~ "button on the right."
#~ msgstr ""
#~ "Test alstublieft of het systeem naar de doelmap kan schrijven door op de "
#~ "rechterkant de \"Test\"-knop in te drukken."

#~ msgid "Test Succeeded !"
#~ msgstr "Test geslaagd!"

#~ msgid "Target directory is writable."
#~ msgstr "Doelmap is beschijfbaar."

#~ msgid ""
#~ "Please change target directory and test writability of the target "
#~ "directory by pressing \"Test\" button on the right."
#~ msgstr ""
#~ "Verander de doelmap en test of het systeem naar de doelmap kan schrijven "
#~ "door op de rechterkant de \"Test\"-knop in te drukken."

#~ msgid ""
#~ "Choose the compression format that you want to apply.\n"
#~ "Default is <b>gzip</b>."
#~ msgstr ""
#~ "Kies het compressie formaat dat je wilt toepassen.\n"
#~ "Default is <b>gzip</b>."

#~ msgid "Use a remote directory (SSH or FTP)"
#~ msgstr "Een map op afstand gebruiken (SSH of FTP)"

#~ msgid "Test"
#~ msgstr "Testen"

#~ msgid ""
#~ "Select the directory in which you have your backups.\n"
#~ "You can use a <b>local</b> or <b>distant</b> directory depending on the "
#~ "plugins you have installed."
#~ msgstr ""
#~ "Selecteer de map met je backups\n"
#~ "Je kunt een locale map of een map op afstand gebruiken, afhankelijk van "
#~ "de plugins die je hebt geinstalleerd."

#~ msgid "Use the default directory : "
#~ msgstr "Gebruik de default map: "

#~ msgid "Use a custom directory : "
#~ msgstr "gebruik een zelfgekozen map: "

#~ msgid "<b>Snapshots directory</b>"
#~ msgstr "<b>Map met snapshots<b>"

#~ msgid "Restore As"
#~ msgstr "Herstellen als"

#~ msgid "Revert As"
#~ msgstr "Terugzetten als"

#~ msgid "Select a snapshot and change its base to another snapshot."
#~ msgstr "Selecteer een snapshot ent het op een andere  snapshot."

#~ msgid "Use default backup directory (/var/backup)"
#~ msgstr "Standaard backupmap gebruiken (/var/backup)"

#~ msgid "A backup run is initiated in the background. The process id is: "
#~ msgstr "Er is een backup gestart in de achtergrond. Het procesnummer is: "

#~ msgid "Use recommended backup settings"
#~ msgstr "Aanbevolen backup-instellingen gebruiken"

#~ msgid "Use custom backup settings"
#~ msgstr "Aangepaste backup-instellingen gebruiken"

#~ msgid ""
#~ "<small>Do backups according to the settings defined on the next pages of "
#~ "this dialog.</small>"
#~ msgstr ""
#~ "<small>Backups maken aan de hand van de instellingen op de volgende "
#~ "schermen van dit dialoogvenster.</small>"

#~ msgid "Regex"
#~ msgstr "Reguliere expressie"

#~ msgid "Mb"
#~ msgstr "Mb"

#~ msgid "precisely"
#~ msgstr "exact"

#~ msgid "at:"
#~ msgstr "om:"

#~ msgid "Day of month:"
#~ msgstr "Dag van de maand:"

#~ msgid "Day of week:"
#~ msgstr "Dag van de week:"

#~ msgid "Hour"
#~ msgstr "Uur"

#~ msgid "Minute"
#~ msgstr "Minuut"

#~ msgid "I didn't find the log file. Please set it up in nssbackup.conf "
#~ msgstr ""
#~ "Het logbestand werd niet gevonden. U kan het instellen in nssbackup.conf "

#~ msgid "Initializing FUSE FILE ACCESS MANAGER !"
#~ msgstr "Bezig met initialiseren van de FUSE FILE ACCESS MANAGER !"

#~ msgid "Problem with '%(file)s' : %(error)s "
#~ msgstr "Probleem met '%(file)s' : %(error)s "

#~ msgid "No directory to backup !"
#~ msgstr "Geen map gevonden om te backuppen!"

#~ msgid ""
#~ "'%(file)s' size is higher than the specified one ( %(filesize)s > %"
#~ "(maxsize)s), it has to be exclude "
#~ msgstr ""
#~ "Bestandsgrootte van '%(file)s' is groter dan de opgegeven grootte ( %"
#~ "(filesize)s > %(maxsize)s), bestand wordt niet opgenomen "

#~ msgid ""
#~ "When setting a username (resp password), password (resp username) is "
#~ "mandatory"
#~ msgstr ""
#~ "Wanneer een gebruikersnaam (respectievelijk wachtwoord) wordt ingesteld, "
#~ "is een wachtwoord (respectievelijk gebruikersnaam) verplicht."

#~ msgid "Set the reciever of this mail"
#~ msgstr "Stel de ontvanger van deze e-mail in"

#~ msgid "Starting snapshot %(name)s "
#~ msgstr "Start snapshot %(name)s "

#~ msgid "Empty regular expression found. "
#~ msgstr "Lege regular expression gevonden "

#~ msgid "Unable to change permissions for "
#~ msgstr "Permissies veranderen is mislukt voor "

#~ msgid "Target '%s' directory is excluded"
#~ msgstr "Doelmap '%s' is uitgesloten"

#~ msgid "Invalid regular expression ('%s')"
#~ msgstr "Ongeldige regular expression ('%s')"

#~ msgid "Choose the SSL option (smtptls=1) to be able to use Cert and Key"
#~ msgstr "Kies de SSL optie (smtptls=1) om Cert and Key te gebruiken"

#~ msgid ""
#~ "Snapshot '%s' is already Full, nothing to do (not changing it to full"
#~ msgstr ""
#~ "Snapshot '%s' is reeds een volledige backup. Niet nodig om volledig te "
#~ "maken."

#~ msgid "NSsbackup configuration error"
#~ msgstr "Configuratiefout NSsbackup"

#~ msgid "Save to file ..."
#~ msgstr "Opslaan naar bestand ..."

#~ msgid "This is a user friendly backup solution for common desktop needs."
#~ msgstr ""
#~ "Dit is een gebruikersvriendelijke backupoplossing die tegemoetkomt aan de "
#~ "noden van de gemiddelde gebruiker."

#~ msgid "'%(file)s' doesn't exist, it has to be exclude "
#~ msgstr "'%(file)s' bestaat niet, bestand wordt niet opgenomen "

#~ msgid ""
#~ "We don't have read access to '%(file)s', it has to be exclude : %(error)s "
#~ msgstr ""
#~ "Kan geen leestoegang krijgen tot '%(file)s', bestand wordt niet "
#~ "opgenomen : %(error)s "

#~ msgid "Terminating FUSE FILE ACCESS MANAGER !"
#~ msgstr "Bezig met afsluiten FUSE FILE ACCESS MANAGER !"

#~ msgid "Got an exception when Pulling '%s' : "
#~ msgstr "Foutmelding bij verkrijgen '%s': "

#~ msgid "Snapshot '%s' is a full . Can't rebase on older snapshot"
#~ msgstr ""
#~ "Snapshot '%s' is een volledige backup. Niet mogelijk om op een ouder "
#~ "snapshot te enten."

#~ msgid "NSsbackup restoration error"
#~ msgstr "NSsbackup herstel-fout"

#~ msgid "Couldn't make a proper backup : "
#~ msgstr "Kon geen goeie backup maken: "

#~ msgid "Non existing directory : %s"
#~ msgstr "Niet bestaande map : %s"

#~ msgid "gtk-remove"
#~ msgstr "gtk-verwijderen"

#~ msgid "gtk-add"
#~ msgstr "gtk-toevoegen"

#~ msgid "gtk-close"
#~ msgstr "gtk-sluiten"

#~ msgid "gtk-edit"
#~ msgstr "gtk-bewerken"

#~ msgid ""
#~ "Please enter the new profile name in the <b>Profile name</b> entry and "
#~ "check the <b>Enable</b> checkbox to enable it."
#~ msgstr ""
#~ "Voer de naam van het nieuwe profiel in in het <b>Profile name</b> vakje "
#~ "en vink <b>Enable</b> aan om te in te schakelen."

#~ msgid ""
#~ "<small>Do not backup your data automatically. Note: this does not prevent "
#~ "doing manual backups.</small>"
#~ msgstr ""
#~ "<small>Geen automatische backups maken. Noot: u kunt nog steeds "
#~ "handmatige backups maken.</small>"

#~ msgid "Test all"
#~ msgstr "Alles testen"

#~ msgid "Time (for admins only)"
#~ msgstr "Tijd (alleen voor admins)"

#~ msgid ""
#~ "Debug is just for debbuging purpose\n"
#~ "Info is the recommended setting"
#~ msgstr ""
#~ "Debug is alleen voor het debuggen. \n"
#~ "Info is de aanbevolen instelling"

#~ msgid "Log settings"
#~ msgstr "Log instellingen"

#~ msgid "Security"
#~ msgstr "Beveiliging"

#~ msgid "Parameters"
#~ msgstr "Parameters"
